recipes/p_astro/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "p_astro" %}
+{% set version = "0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3df0ab549f07f9de1c9e6b153e4cad65aedaca247d53665a197e78cc8ad80d2a
+  patches:
+    # remove unncessary build dependency
+    - no-pytest-runner.patch
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - python >=3
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - ligo.p_astro
+  requires:
+    - pytest
+
+about:
+  home: http://gracedb.ligo.org
+  doc_url: https://lscsoft.docs.ligo.org/p-astro/
+  license: GPLv3
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Let’s make sure to submit the license upstream.

Reply:
This has been reported upstream, but the repo is not publicly viewable.